core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -604,5 +623,22 @@ public static void validate() {
     checkTieredLocality();
   }
 
+  /**
+   * @return a list of worker-related configurations
+   */
+  public static List<ConfigProperty> getWorkerConfiguration() {
+    List<ConfigProperty> configInfoList = new ArrayList<>();
```

Comment:
Can we write a method that returns the set of all property keys? Then this would be 
```java
return getAllPropertyKeys().stream()
  .filter(key -> key.getScope().contains(Scope.WORKER))
  .map(key -> new ConfigProperty().setName(key.getName()).setValue(get(key))
      .setSource(getFormattedSource(key))).collect()
```

Reply:
Changed to : 
```
  public static List<ConfigProperty> getWorkerConfiguration() {
    return toMap().keySet().stream().map(PropertyKey::fromString)
        .filter(key -> key.getScope().contains(Scope.WORKER))
        .map(key -> new ConfigProperty().setName(key.getName())
            .setValue(get(key)).setSource(getFormattedSource(key)))
        .collect(Collectors.toList());
  }
```

Add one ```.map(PropertyKey::fromString) ```so that we do not need to write a separate method.